src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -88,27 +72,28 @@ var DOMRenderer = ReactFiberReconciler({
     eventsEnabled = null;
   },
 
-  updateContainer(container : Container, children : HostChildren<Instance | TextInstance>) : void {
-    // TODO: Containers should update similarly to other parents.
-    container.innerHTML = '';
-    recursivelyAppendChildren(container, children);
-  },
-
   createInstance(
     type : string,
     props : Props,
-    children : HostChildren<Instance | TextInstance>,
     internalInstanceHandle : Object
   ) : Instance {
-    const root = document.body; // HACK
+    const root = document.documentElement; // HACK
 
     const domElement : Instance = createElement(type, props, root);
     precacheFiberNode(internalInstanceHandle, domElement);
-    recursivelyAppendChildren(domElement, children);
-    setInitialProperties(domElement, type, props, root);
     return domElement;
   },
 
+  appendInitialChild(parentInstance : Instance, child : Instance | TextInstance) : void {
```

Comment:
will this only be called on the initial mount? or also when new children get's appended on a already mounted node?

will appendChild or appendInitialChild handle: `["Foo"]` -> `["Foo", "Bar"]` ?